todo_app/tests/mock_responses.py

```diff
@@ -0,0 +1,991 @@
+class MockResponses():
+    def __init__(self):
+        pass
+
+
+    @property
+    def trello_user(self):
+        return 'bobinaki'
```

Comment:
This "trello_user" is not part of the mock response... it's part of your app config. And it's not just test config, it actually has to match the real (and uncommitted) value in trello.config in order for the test to pass. Which is not great.
Similarly for the board_name has to match the real board name in trello.config because your app checks that `board.name == board_name:`

Separate to getting the tests to pass, you currently read the board_id from an environment variable but everything else from your trello.config file, which is confusing and causes extra complication.

The exercise instructions explain how to override the config when running tests but assumes you were using environment variables. With a trello.config file, things become more complicated.

You should either switch to just using environment variables or if you feel attached to the trello.config file, you can keep it but things get a bit more complicated:
You should have multiple versions of it, where the test version is added to git. Then the test should tell TrelloAPI to use the test one, either via a single env var or even by overriding TrelloAPI.CREDS_CONFIG_FILE, but that can cause other issues.
And in later modules you will need to do a little extra work to create the config file when running it in a container or on a server.

Reply:
Removed completely dependency on trello.config and deleted it. All now across the project, is dependenct on environment vars based on .env / .env.test